src/app/models/function.rb

```diff
@@ -1,5 +1,5 @@
 class Function < ApplicationRecord
-  validates :name, presence: true, uniqueness: { scope: :user_id }, format: { with: /\A[0-9a-z_]+\z/i }
+  validates :name, presence: true, uniqueness: { scope: :user_id }
```

Comment:
[R] I prefer to be strict about the format, we shouldn't remove the format from here

Reply:
My point was that RubyVM parser will explicitly enforce the formatting so I thought this would add unnecessary redundancy.